api/environments/dynamodb/dynamodb_wrapper.py

```diff
@@ -0,0 +1,80 @@
+import typing
+
+import boto3
+from boto3.dynamodb.conditions import Key
+from django.conf import settings
+from django.core.exceptions import ObjectDoesNotExist
+
+
+class DynamoIdentityWrapper:
+    _table = None
```

Comment:
I think this should be done in `__init__`. I think this could cause some strange behaviour otherwise.

Reply:
ah that can make mocking a bit more pain

Reply:
I'd rather the mocking was a pain than the actual production logic... As far as I'm aware, you shouldn't really set anything mutable as an attribute directly on the class. Couldn't you extract the logic to a `get_dynamo_table` function that you can subsequently mock?